bower.json

```diff
@@ -25,8 +25,8 @@
     "api"
   ],
   "dependencies": {
-    "angular": "~1.2",
-    "angular-ext-q": "~1.0",
+    "angular": "1.2 - 1.5",
+    "angular-ext-q": "1.1.0",
```

Comment:
@Minglee01 I don't think we should pin this to `1.1.0`. Using `~1` will still match `1.1.0` but also update to any other `1.x` version.